src/Microsoft.AspNet.Mvc.Core/ModelBinding/Validation/DefaultObjectValidator.cs

```diff
@@ -165,32 +159,41 @@ private void MarkPropertiesAsSkipped(string currentModelKey, ModelMetadata metad
             }
         }
 
+        private IList<IModelValidator> GetValidators(IModelValidatorProvider provider, ModelMetadata metadata)
+        {
+            var validatorProvider = provider;
+            var validatorProviderContext = new ModelValidatorProviderContext(metadata);
+            validatorProvider.GetValidators(validatorProviderContext);
+            return validatorProviderContext.Validators;
+        }
+
         private bool ValidateProperties(
```

Comment:
Lousy name since this method now handles all child nodes -- whether elements or properties.